source/Nevermore.IntegrationTests/Model/StringTinyTypeIdTypeHandler.cs

```diff
@@ -0,0 +1,37 @@
+﻿#nullable enable
+using System;
+using System.Data;
+using System.Data.Common;
+using Nevermore.Advanced.TypeHandlers;
+
+namespace Nevermore.IntegrationTests.Model
+{
+    class StringTinyTypeIdTypeHandler<T> : ITypeHandler where T : TinyType<string>
```

Comment:
I would avoid referencing `TinyType` name anywhere in Nevermore because this might cause quite a bit of confusion. What about `CustomIdType` instead?

Reply:
yep 👍